second-analysis-steps/filter-in-trees.md

```diff
@@ -53,25 +34,20 @@ from Configurables import CombineParticles
 from PhysSelPython.Wrappers import Selection, DataOnDemand
 
 Pions = DataOnDemand('Phys/StdAllNoPIDsPions/Particles')
-rho = CombineParticles('rho_particles',
-                       DecayDescriptor=['rho(770)0 -> pi+ pi-'],
-                       CombinationCut="ADAMASS('rho(770)0') < 300*MeV",
+kst = CombineParticles('kst_particles',
+                       DecayDescriptor=['[K*(892)0 -> K+ pi-]cc'],
```

Comment:
I think this should be `CC` instead of `cc`.

I've just fixed it, if I'm wrong let me know and I'll change it back.

Reply:
As far as I remember, all the CombineParticles and selections in general should take "cc", otherwise it crashes. (See the Build you decay - Historical Approach lesson).
Let someone confirm this.

Reply:
Where do you see that? I could only find the lowercase variant in the context of a single particle like [`"[ [B0]cc => K+ pi- ]CC"`](https://twiki.cern.ch/twiki/bin/view/LHCb/FAQ/LoKiNewDecayFinders#How_to_specify_charge_conjugated).

Reply:
Simply check running both versions of code, one should crash.

Reply:
Yes, I'll try that when I get to CERN.